tasks/ssl.yml

```diff
@@ -22,8 +22,9 @@
       remote_src: "{{ sensu_ssl_deploy_remote_src }}"
       group: "{{ sensu_group_name }}"
       dest: "{{ sensu_config_path }}/ssl/{{ item.dest }}"
+      mode: " {{ item.perm }}"
     with_items:
-      - {src: "{{ sensu_ssl_client_cert }}", dest: cert.pem}
-      - {src: "{{ sensu_ssl_client_key }}", dest: key.pem}
+      - {src: "{{ sensu_ssl_client_cert }}", dest: cert.pem , perm: "0644" }
```

Comment:
I think this should also be `0640` as well which matches what I have in my setup:
```
sudo bash -c 'ls -la /etc/sensu/ssl/'
total 16
drwxr-x--- 2 root sensu 4096 Jan 26 16:28 .
drwxr-x--- 8 root sensu 4096 Jan 26 16:28 ..
-rw-r----- 1 root sensu 1054 Nov 13 22:54 cert.pem
-rw-r----- 1 root sensu 1675 Nov 13 22:54 key.pem
```

Reply:
yes, maybe...

Reply:
Yeah, I agree with @majormoses, I too use `0640` in my environment without issues. I think it makes sense as a default here. If anyone hits issues with that, we can always extract it into a variable as well.